roles/container-engine/cri-o/templates/crio.conf.j2

```diff
@@ -280,6 +280,7 @@ network_dir = "/etc/cni/net.d/"
 
 # Paths to directories where CNI plugin binaries are located.
 plugin_dirs = [
-	"/usr/libexec/cni",
-	"/opt/cni/bin/",
+{% for dir in crio_cni_plugin_dirs %}
```

Comment:
`/usr/libexec/cni`

is not listed in the plugin dirs. Is that on purpose ?

Reply:
> 
> 
> `/usr/libexec/cni`
> 
> is not listed in the plugin dirs. Is that on purpose ?

Yes but I could be wrong here. Here's what I found on my system:
* package cri-o depends on package containernetworking-plugins that installs a bunch of cni plugins into /usr/libexec/cni

* kubespray configures calico to install into /opt/cni/bin

* since /usr/libexec/cni is listed first in crio config and due to the bug in crio it will never find calico

What do you think I should do with this patch? One way would be to just make it configurable with the same list of cni dirs. Can /usr/libexec/cni ever be used by a kubespray install?

Reply:
On my environment here I have all the stuff on `/opt/cni/bin`  so I think it should be fine

Reply:
Yes it is the same thing (containernetworking-plugins) installed both by RPM dependency due to rio and by kubespray. The RPM is older then what kubespray install. I think it should be safe to remove the path as proposed.

Another problem is the mix of installation methods, rpms, tarballs